src/Drush/Commands/Commands.php

```diff
@@ -9,32 +9,27 @@
 use Drush\Commands\DrushCommands;
 use Drush\Exceptions\CommandFailedException;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drush\Commands\AutowireTrait;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
 
 /**
  * Drush commands.
  */
 final class Commands extends DrushCommands {
 
+  use AutowireTrait;
+
   /**
    * Commands constructor.
    */
   public function __construct(
+    #[Autowire(service: \Drupal\azure_ad_delta_sync\Controller::class)]
     private readonly ControllerInterface $controller,
+    #[Autowire(service: \Drupal\azure_ad_delta_sync\UserManager::class)]
```

Comment:
```suggestion
    #[Autowire(service: UserManager::class)]
```

Why doesn't Rector do/suggest this?